coredns/go.mod

```diff
@@ -20,6 +20,15 @@ require (
 	sigs.k8s.io/mcs-api v0.1.0
 )
 
+// Pin K8s or now until github.com/coredns/coredns bumps to 0.27.*
+replace (
+	k8s.io/api => k8s.io/api v0.26.3
+	k8s.io/apimachinery => k8s.io/apimachinery v0.26.3
+	k8s.io/client-go => k8s.io/client-go v0.26.3
+	k8s.io/utils => k8s.io/utils v0.0.0-20221128185143-99ec85e7a448
```

Comment:
This shouldn’t be necessary, `k8s.io/utils` is maintained in a backwards-compatible fashion.